libs/core/src/lib/shellbar/shellbar-actions/shellbar-actions.component.scss

```diff
@@ -0,0 +1,19 @@
+.fd-shellbar__action-fullwidth {
```

Comment:
This should be BEM friendly and also in fundamental-styles but I think let's get this merged and handle it in a separate issue